src/main/java/bibliotecame/back/Review/ReviewController.java

```diff
@@ -0,0 +1,80 @@
+package bibliotecame.back.Review;
+
+import bibliotecame.back.Book.BookModel;
+import bibliotecame.back.Book.BookService;
+import bibliotecame.back.Copy.CopyModel;
+import bibliotecame.back.Loan.LoanModel;
+import bibliotecame.back.User.UserModel;
+import bibliotecame.back.User.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/review")
+public class ReviewController {
+
+    private final BookService bookService;
+    private final ReviewService reviewService;
+    private final UserService userService;
+
+    @Autowired
+    public ReviewController(ReviewService reviewService, UserService userService, BookService bookService) {
+        this.reviewService = reviewService;
+        this.userService = userService;
+        this.bookService = bookService;
+    }
+
+    @GetMapping("/{id}")
+    public ResponseEntity<ReviewModel> getReviewModel(@PathVariable Integer id){
+        //Este método será ampliado y modificado por quien corresponda cuando se haga la US "Visualizar reseña"
+        return new ResponseEntity<>(this.reviewService.findReviewById(id), HttpStatus.OK);
+    }
+
+    @PostMapping("/create/{bookId}")
+    public ResponseEntity<ReviewModel> createReview(@Valid @RequestBody ReviewModel reviewModel, @PathVariable Integer bookId){
```

Comment:
The endpoint should only receive:
- The book id (on the URL)
- The value of the review (on the body)
- Optionally the description of the review (also in the body)

The user we must take it from the logged user and I don't know what is the `'active'` boolean.